### PR TITLE
add mission statement to About

### LINKED ISSUE #363: add mission statement
At the Business Meeting on 2024-02-26 the @MDAnalysis/coredevs agreed to move forward with the "version 1" draft for a Mission Statement.

Note that this statement will be used in the future decide on directions to take, to prioritize efforts, and generally guide the project at a high level. Therefore, it is important that everyone who takes an interest in the direction of MDAnalysis should feel free to voice their opinion.

### PR DESCRIPTION
**Pull request by orbeckst**
Closes #363 

This PR
- adds a mission statement under **About the MDAnalysis Project** to the top of the [About](https://www.mdanalysis.org/about/) page ("version 1" draft, as agreed at Business Meeting on 2024-02-26)
- asks @MDAnalysis/coredevs to review and alter the language as needed
- invites anyone interested in the direction that MDAnalysis takes to comment
- asks @MDAnalysis/coredevs to ultimately approve the final mission statement

This Mission Statement is not set in stone and should be periodically revisited and updated. However, it seems unlikely that such updates will happen frequently so you should consider this language to have a long half-life (i.e., it will guide decisions for years to come).

### CODE REVIEW

**Review by hmacdope — APPROVED** [review]

**Review by RMeli — APPROVED** [review]

**Review by richardjgowers — APPROVED** [review]

**Review by jennaswa — APPROVED** [review]

**Comment by orbeckst** [comment]
@fiona-naughton @lilyminium @micaela-matta @tylerjereddy @yuxuanzhuang do you have any comments about the mission statement or are you ready to approve it?

File: about.md [hunk 1]
```diff
@@ -6,6 +6,26 @@ title: About MDAnalysis
 <img src="{{ site.baseurl }}/public/mdanalysis-logo_square.png"
 style="float: right" alt="MDAnalysis" width="30%"/>
 
+## About the MDAnalysis Project
```
Comment (orbeckst) [inline about.md:9]: Should this be changed to "Mission" ?

Repeating "About" in two headings looks a bit awkward.

Comment (orbeckst) [reply about.md:9]:
```suggestion
## Mission
```

Comment (lilyminium) [reply about.md:9]: I like the change

**Review by lilyminium — APPROVED** [review]

File: about.md [hunk 2]
```diff
@@ -6,6 +6,26 @@ title: About MDAnalysis
...
+
+The MDAnalysis [Community](#community) is interested in all facets of **working
+with data in the molecular sciences**. We welcome everyone. **We all follow our
+[Code of Conduct] and strive to create an environment that is welcoming to
+all.** Our primary purpose is to produce software that scientists in academia and
+industry will trust to use in their research.
+
+We develop and maintain projects related to the broader goal of processing and
+analyzing data in the molecular sciences. We aim to empower users/developers to
```
Comment (tylerjereddy) [inline about.md:18]: Here and above "molecular sciences" seems incredibly broad. I suspect someone could propose a molecular science functionality we'd consider out of scope.

That said, I don't really care too much, the general spirit seems correct.

**Review by tylerjereddy — APPROVED** [review]

File: about.md (hunk 2 above)
Comment (orbeckst) [reply about.md:18]: That's a fair point... I guess someone could think about NMR data or cryo-ET. 

Are we going to make our life more difficult by keeping it fairly broad? Or does it potentially open up opportunities?

Comment (hmacdope) [reply about.md:18]: In OMSF we often use "computational molecular sciences" to narrow the scope a bit.

**Review by yuxuanzhuang — APPROVED** [review]

File: about.md (hunk 2 above)
Comment (orbeckst) [reply about.md:18]: Sounds good to me. do you want to edit accordingly?

Comment (hmacdope) [reply about.md:18]: Yes please.

File: about.md [hunk 3]
```diff
@@ -6,6 +6,26 @@ title: About MDAnalysis
 <img src="{{ site.baseurl }}/public/mdanalysis-logo_square.png"
 style="float: right" alt="MDAnalysis" width="30%"/>
 
+## Mission
+
+The MDAnalysis [Community](#community) is interested in all facets of **working
+with data in the molecular sciences**. We welcome everyone. **We all follow our
```
Comment (orbeckst) [inline about.md:12]:
```suggestion
with data in the computational molecular sciences**. We welcome everyone. **We all follow our
```

File: about.md [hunk 4]
```diff
@@ -6,6 +6,26 @@ title: About MDAnalysis
...
+[Code of Conduct] and strive to create an environment that is welcoming to
+all.** Our primary purpose is to produce software that scientists in academia and
+industry will trust to use in their research.
+
+We develop and maintain projects related to the broader goal of processing and
+analyzing data in the molecular sciences. We aim to empower users/developers to
```
Comment (orbeckst) [inline about.md:18]:
```suggestion
analyzing data in the computational molecular sciences. We aim to empower users/developers to
```

**Review by orbeckst — COMMENTED** [review]
Narrowing the scope, following @tylerjereddy and @hmacdope 's suggestions.

**Comment by orbeckst** [comment]
ping @fiona-naughton @micaela-matta — once you laid 👀 on this PR and ✅ it (or ❌ it and 📝 and _then_ ✅ it) , we will finally have our very own mission statement!

**Review by fiona-naughton — APPROVED** [review]

**Review by micaela-matta — APPROVED** [review]

**Comment by orbeckst** [comment]
Thanks everyone!

**Pull request merged**